### ISSUE TITLE
[stable/prometheus-operator] introduction of metrics_path breaks alert rules

### ISSUE THREAD

**Issue by smueller18** [helm/charts#20224]
**Describe the bug**
In https://github.com/helm/charts/commit/6ba801353386aba170d803534f1051f81b784f57#diff-f94f3489fb0810db0ffcc8d410026981, `metrics_path` was added as filter in many expressions. This change breaks the related rules, because `metrics_path` is not present in prometheus metrics.
There is already the comment https://github.com/helm/charts/commit/71d5b06c1547ed2bf7fa3df17873f1f3f27bf7ef#r36782310 which asks about this change but has not get answered yet.

**Version of Helm and Kubernetes**:
```
$ helm version
version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}
$ kubectl version
serverVersion:
  buildDate: "2019-12-07T21:12:17Z"
  compiler: gc
  gitCommit: 70132b0f130acc0bed193d9ba59dd186f0e634cf
  gitTreeState: clean
  gitVersion: v1.17.0
  goVersion: go1.13.4
  major: "1"
  minor: "17"
  platform: linux/amd64
```

**Which chart**:
stable/prometheus-operator 8.5.10

**What happened**:
Alert `KubeletDown` is firing.
`absent(up{job="kubelet",metrics_path="/metrics"} == 1)`

**What you expected to happen**:
Alert `KubeletDown` is not firing.
`absent(up{job="kubelet"} == 1)`

**How to reproduce it** (as minimally and precisely as possible):
Install kubernetes v1.17.0 and install this helm chart. 

**Anything else we need to know**:

The query `up{job="kubelet"}` shows the elements like the following:
`up{endpoint="https-metrics",instance="192.168.1.11:10250",job="kubelet",namespace="kube-system",node="node1.example.org",service="monitoring-kubelet"}`

**Comment by seanson** [comment]
This should be fixed by #20194

**Comment by smueller18** [comment]
I can confirm that the change in #20194 closes the described bug.
While checking the alerts, I found out, that the alert `KubeletDown` is defined twice, first in `kubernetes-absent.yaml` and second `kubernetes-system-kubelet.yaml`. I will open a MR for fixing this.
